### PR TITLE
LPS-159111 Test Automation Creation | master

### PR DESCRIPTION
**Pull request by ozysouza**
https://issues.liferay.com/browse/LPS-159111

cc @david-gutierrez-mesa 
cc @Tim-Cao

### CODE REVIEW

**Comment by liferay-echo** [comment]
Thank you for submitting this PR. 👏

Make sure it follows the [Echo Pull Request Template](https://gist.github.com/liferay-echo/5a4455285ff1800f9da146816d85e67d#file-pull-request-template-md) if you haven't already done so. 

Learn more about it [here](https://liferay.atlassian.net/wiki/spaces/ENGECHO/pages/2027455014/Echo+Pull+Request+Etiquette).

**Comment by Tim-Cao** [comment]
Hi @ozysouza, could you migrate cases to ContentPagesWithSelectionPanel.testcase?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase [hunk 1]
```diff
@@ -1145,6 +1145,45 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-137155. The page creator can change permissions of content page via header ellipsis icon at edite mode."
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase:1148]: Check typo issue

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase [hunk 2]
```diff
@@ -1145,6 +1145,45 @@ definition {
...
+	@priority = "3"
+	test RemoveViewGuestPermissionViaHeaderEllipsisIcon {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase:1150]: RemoveViewPermissionForGuestViaHeaderEllipsisIcon

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase [hunk 3]
```diff
@@ -303,6 +303,68 @@ definition {
...
+	@description = "This is a test for LPS-137155. The page creator can configure content page via header ellipsis icon at edit mode."
+	@priority = "3"
+	test ConfigureViaHeaderEllipsisIcon {
+		task ("Given a page designer has content page in pages admin") {
+			PagesAdmin.openPagesAdmin(siteURLKey = "test-site-name");
+
+			PagesAdmin.viewPageItemPresent(pageName = "Test Content Page Name");
+		}
+
+		task ("When the page designer configures the page via header ellipsis icon at edit mode") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase:315]: When the page designer renames page via header ellipsis icon at edit mode

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase [hunk 4]
```diff
@@ -303,6 +303,68 @@ definition {
...
+				siteName = "Test Site Name");
+
+			LexiconEntry.gotoHeaderMenuItem(menuItem = "Configure");
+
+			PortletEntry.inputName(name = "Test Page Name Edited");
+
+			PortletEntry.save();
+		}
+
+		task ("Then the page designer can see edited page name in pages admin") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPages.testcase:327]: Then the page designer can see new page name in pages admin

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase [hunk 5]
```diff
@@ -1145,6 +1145,45 @@ definition {
...
+				addOrRemoveViewPermission = "Remove",
+				permissionsKeyList = "INLINE_PERMISSIONS_VIEW_CHECKBOX",
+				roleTitle = "guest");
+
+			IFrame.closeFrame();
+
+			PageEditor.publish();
+		}
+
+		task ("Then a non logged user should not see the content page") {
```
Comment (Tim-Cao) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase:1174]: Then a guest user cannot access to the content page

**Review by Tim-Cao — COMMENTED** [review]
Hi @ozysouza, would you mind update according to comment? Thanks

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase (hunk 1 above)
Comment (ozysouza) [reply portal-web/test/functional/com/liferay/portalweb/tests/enduser/wem/layout/contentpage/ContentPagesWithPermissions.testcase:1148]: Thanks for the heads up!

**Comment by ozysouza** [comment]
> Hi @ozysouza, would you mind update according to comment? Thanks

Hi @Tim-Cao, PR updated! Could you have a look at? Thanks!

**Comment by Tim-Cao** [comment]
LGTM!

**Comment by Tim-Cao** [comment]
ci:forward

**Pull request closed without merging**